plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -176,7 +176,7 @@ local DoubleKeyValuePage = InputContainer:new{
     show_page = 1,
     text_input = "",
     pages = 1,
-    goodreadersKey = "",
```

Comment:
Um, wait a second, both of your comments are requesting the change that's already been made! =P